#! /bin/sh -e

# DP:

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
       echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0



Index: python2.5-2.5.2/setup.py
===================================================================
--- python2.5-2.5.2.orig/setup.py	2009-05-04 16:28:10.000000000 -0400
+++ python2.5-2.5.2/setup.py	2009-05-04 16:28:29.000000000 -0400
@@ -796,7 +796,7 @@
                                         sqlite_version, MIN_SQLITE_VERSION)
                 elif sqlite_setup_debug:
                     print "sqlite: %s had no SQLITE_VERSION"%(f,)
-        
+	sqlite_incdir = None       
 	if sqlite_incdir:
             sqlite_dirs_to_check = [
                 os.path.join(sqlite_incdir, '..', 'lib64'),
